### PR TITLE
fix(Inline): improve fullWidth case

### PR DESCRIPTION
**Pull request by pladaria**
This branch against webapp:
https://github.com/Telefonica/webapp/actions/runs/7667071028

A single test failed, which is expected:
header-cards-single-screenshot-test-tsx-header-cards-single-two-buttons-brand-vivo-br-in-mobile-ios-1-diff

### CODE REVIEW

File: src/dialog.tsx [hunk 1]
```diff
@@ -34,6 +34,8 @@ interface BaseDialogProps {
     destructive?: boolean;
     /** @deprecated this does nothing */
     forceWeb?: boolean;
+    /** @deprecated this does nothing */
+    showCancel?: boolean;
```
Comment (pladaria) [inline src/dialog.tsx:38]: unrelated, to keep dialog types compatible

File: src/__tests__/dialog-test.tsx [hunk 2]
```diff
@@ -72,15 +72,17 @@ test('throws when we try to stack dialogs', async () => {
...
 
 test("throws when we don't instantiate theme", async () => {
     render(<TestComponent />);
 
-    expect(() => savedAlert(alertProps)).toThrow(
-        'Tried to show a dialog but the DialogRoot component was not mounted'
-    );
+    expect(() => {
+        act(() => savedAlert(alertProps));
+    }).toThrow('Tried to show a dialog but the DialogRoot component was not mounted');
```
Comment (pladaria) [inline src/__tests__/dialog-test.tsx:85]: this fixes the unstable test

File: src/inline.css.ts [hunk 3]
```diff
@@ -49,44 +45,34 @@ export const inline = style({
...
-    },
-]);
+export const fullWidth = style({
+    /**
+     * @deprecated - this should be "flex"
+     * Once changed, the usages that depend on this behavior should be migrated to use <Grid>
+     * See "inline-cases" private story
+     */
+    display: 'grid',
+});
```
Comment (atabel) [inline src/inline.css.ts:61]: I don't understand this change. With the previous code we were using flex in browsers that don't support grid

File: src/inline.css.ts [hunk 4]
```diff
@@ -49,44 +45,34 @@ export const inline = style({
...
-        display: ['inline-flex', 'inline-grid'],
-        '@supports': {
-            [supportsFlexGap]: {
-                display: 'inline-flex',
-            },
-        },
-    },
-]);
+export const noFullWidth = style({
+    display: 'inline-grid',
```
Comment (marcoskolodny) [inline src/inline.css.ts:69]: Same thing that [Abel commented](https://github.com/Telefonica/mistica-web/pull/1007/files#r1470741091). Why removing the fallback to inline-flex if grid is not supported?

As far as I know, inline-grid is not supported in all the browsers that webapp supports (https://caniuse.com/?search=inline-grid)

File: src/inline.css.ts (hunk 3 above)
Comment (pladaria) [reply src/inline.css.ts:61]: Added a fallback to flex

File: src/inline.css.ts (hunk 4 above)
Comment (pladaria) [reply src/inline.css.ts:69]: added fallback

**Review by atabel — APPROVED** [review]

**Pull request merged**